### ISSUE TITLE
This library does not support bitcode

### ISSUE THREAD

**Issue by dreampiggy** [SDWebImage/SDWebImageHEIFCoder#2]
Thanks for creating this library I am using this lib. since five years its great.

Now I am getting issue for .heic image format file, SDWebImage not load images for .heic format . I am used SDWebImageHEICCoder , its load file successfully but build issue while archive due to bitcode enable yes. if disable bitcode than its working fine.

Can you please check this.

**Comment by dreampiggy** [comment]
This is because the vendored static library `libx265.a`, `libde265.a` was built without bitcode support. If your project need bitcode, all source-code, static library, dynamic library (including yourself and third-party library) should be built with bitcode.

@visionsIT 
Does your application need bitcode support ? If so, I can provide a approach to use a Pod subspec for bitcode support. With the new built static library.

**Comment by dreampiggy** [comment]
Done. I'll update the new static library built with bitcode support. And also introduce the watchOS platform support as well. :)

```bash
➜  ios otool -l libde265.a | grep bitcode
  sectname __bitcode
```

And also, I'm trying another solution (not immediatelly available, but it's a better one), which will drop the vendor binary and use a standalone repo for the dependency `libheif`, `libde265`, `x265`.

Previously it's hard to write a wrapper, because all of them are C/C++ library with GNU toolchain without Xcode build support. But we can create a repo like [libde265-Xcode](https://github.com/SDWebImage/libde265-Xcode) and move their config into Xcode config.

This solution can make that the end-user to use CocoaPods with source files,  and even Carthage support.

**Comment by dreampiggy** [comment]
@visionsIT Solved. 

For 4.x compatible branch, fixed by [0.1.2 release](https://github.com/SDWebImage/SDWebImageHEIFCoder/releases/tag/0.1.2). If you're using 4.x SDWebImage, please upgrade the Podfile.

For 5.x master branch, fixed by [0.2.1 release](https://github.com/SDWebImage/SDWebImageHEIFCoder/releases/tag/0.2.1). You need SDWebImage 5.x to use the coder plugin.

**Comment by visionsIT** [comment]
thanks @dreampiggy
